src/Row.tsx

```diff
@@ -28,7 +29,6 @@ function Row<R, SR = unknown>(
     height,
     onRowChange,
     selectCell,
-    selectRow,
     'aria-rowindex': ariaRowIndex,
     'aria-selected': ariaSelected,
     ...props
```

Comment:
Do these two props actually need to be destructured?

Reply:
no, removed